### PR TITLE
Add configuration for running celery beat

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: openshift/packit-service-beat.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+          name: packit-worker:{{ deployment }}
+  replicas: 1
+  strategy:
+    type: Recreate
+---
+# https://docs.openshift.com/container-platform/3.11/dev_guide/managing_images.html#writing-image-stream-definitions
+kind: ImageStream
+apiVersion: v1
+metadata:
+  name: packit-worker
```
Comment (jpopelka) [inline openshift/packit-service-beat.yml.j2:113]: This ImageStream is already created by the `packit-service-worker.yml.j2` and the beat will use it too.

File: openshift/packit-service-beat.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+              mountPath: /home/packit/.config
+          command:
+            - "/usr/bin/run_worker.sh"
+          resources:
+            requests:
+              memory: "100Mi"
+              cpu: "50m"
+            limits:
+              memory: "200Mi"
+              cpu: "100m"
```
Comment (jpopelka) [inline openshift/packit-service-beat.yml.j2:94]: The beat doesn't need so much memory. If I understand it correctly it'll do nothing most of the time.
```suggestion
            limits:
              memory: "80Mi"
              cpu: "50m"
```

File: openshift/packit-service-beat.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+              valueFrom:
+                secretKeyRef:
+                  key: database-user
+                  name: postgres-secret
+            - name: POSTGRESQL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: database-password
+                  name: postgres-secret
+            - name: POSTGRESQL_DATABASE
```
Comment (jpopelka) [inline openshift/packit-service-beat.yml.j2:72]: I'm just wondering if we need to set all those variables from all the secrets.
The beat should IMHO not need any of those, but I'll leave this on you to try.

File: vars/stg_template.yml [hunk 4]
```diff
@@ -46,6 +46,8 @@ api_key: ""
 
 # with_flower: true
 
+# with_beat: true
```
Comment (jpopelka) [inline vars/stg_template.yml:49]: add it to `vars/template.yml` as well, please

File: openshift/packit-service-beat.yml.j2 [hunk 5]
```diff
@@ -0,0 +1,125 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
```
Comment (lachmanfrantisek) [inline openshift/packit-service-beat.yml.j2:3]:
```suggestion
# Copyright (c) 2020 Red Hat, Inc.
```

File: openshift/packit-service-beat.yml.j2 (hunk 2 above)
Comment (lbarcziova) [reply openshift/packit-service-beat.yml.j2]: When I was trying to run it with this configuration, beat had problems to start so I made the limits higher, is it okay?

Comment (jpopelka) [reply openshift/packit-service-beat.yml.j2]: ok

File: openshift/packit-service-beat.yml.j2 (hunk 3 above)
Comment (lbarcziova) [reply openshift/packit-service-beat.yml.j2:72]: As the packit_service.worker.tasks is called and it imports the app from cerelizer.py, I think the redis and postgres env vars are needed.

**Pull request merged**